src/database/sql_database.cc

```diff
@@ -1264,14 +1264,17 @@ fs::path SQLDatabase::buildContainerPath(int parentID, const std::string& title)
     return path;
 }
 
-void SQLDatabase::addContainerChain(std::string virtualPath, const std::string& lastClass, int flags, int lastRefID, int* containerID, std::deque<int>& updateID, const std::vector<std::pair<std::string, std::string>>& lastMetadata)
+bool SQLDatabase::addContainer(int parentContainerId, std::string virtualPath, const std::shared_ptr<CdsContainer>& cont, int* containerID)
 {
-    log_debug("Adding container Chain for path: {}, lastRefId: {}, containerId: {}", virtualPath.c_str(), lastRefID, *containerID);
+    log_debug("Adding container Chain for path: {}, lastRefId: {}, containerId: {}", virtualPath.c_str(), cont->getRefID(), *containerID);
+
+    if (parentContainerId == INVALID_OBJECT_ID)
```

Comment:
I personally think this should bail out instead, you have to explicitly set a root attachment.

Reply:
The only call comes from `ContentManager`, so this can be changed.